Example-Crypto-REXX/RXCRYPT1.rexx

```diff
@@ -0,0 +1,2399 @@
+/*                                                                     */
+/***********************************************************************/
+PrepLPARAttributes:
+
+drop LPARAttribute.
+
+attrI = 1
+LPARAttribute.attrI = 'crypto-activity-cpu-counter-authorization-control'
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'assigned-crypto-domains'
+LPARAttribute.attrI.0 = 1
+attrI = attrI + 1
+LPARAttribute.attrI = 'assigned-cryptos'
+LPARAttribute.attrI.0 = 1
+
+/* total number of LPAR attributes to query
+   NOTE: this is not the total number of properties in the csv file
+         because each complex property may result in one or more
+         properties
+*/
+LPARAttribute.0 = attrI
+
+return  /* end function */
+
+/*************************************************/
+/* Function:  foundJSONvalue()                   */
+/* Return true if the content is not empty       */
+/*************************************************/
+foundJSONValue:
+parse arg jsonValueArg
+
+if jsonValueArg = '' | jsonValueArg = '(not found)' then
+  return FALSE
+
+return TRUE
+
+/*************************************************/
+/* Function:  EntryArrow                         */
+/* Print entry arrow '--------------->'          */
+/*************************************************/
+EntryArrow:
+say
+say '--------------->'
+return
+
+/*************************************************/
+/* Function:  ExitArrow                          */
+/* Print entry arrow '<---------------'          */
+/*************************************************/
+ExitArrow:
+say '<---------------'
+say
+return
+
+/*************************************************/
+/* Function:  WriteToFile()                      */
+/* This will write out the REXXWRT stem contents */
+/*************************************************/
+WriteToFile:
+parse arg file
+
+rc = 0
+if ISVREXX then
+  do
+    "ALLOC F(MYOUTD) DSN("||file||") SHR"
+    Address MVS "EXECIO * DISKW MYOUTD (STEM REXXWRT. FINIS"
+    return_code = RC
+    "FREE F(MYOUTD)"
+  end
+else
+  do
+    Address TSO
+    "ALLOC F(MYOUTD) DSN("||file||") SHR REU"
+    "EXECIO * DISKW MYOUTD (STEM REXXWRT. FINIS"
+    return_code = RC
+    "FREE F(MYOUTD)"
+  end
+
+RC = return_code
+
+if RC <> 0 then
+  do
+    errMsg = '** fatal error, rc=('||RC,
+             ||') encountered trying to write out content **'
+    return fatalError(errMsg)
+  end
+
+return  rc /* end function */
+
+/*************************************************/
+/* Function:  VerifyDataSet()                    */
+/*                                               */
+/* Verify the data set identified by the         */
+/* userDataSet argument exists.                  */
+/*                                               */
+/* Return 0 if data set exists,                  */
+/*          otherwise a non zero                 */
+/*************************************************/
+VerifyDataSet:
```

Comment:
The indentation of the function should be consistent across the file's functions

Reply:
Fixed.